src/app/treasury/TreasuryContext.tsx

```diff
@@ -65,6 +65,7 @@ const TreasuryContextProvider = ({ children }: Props) => {
   const bucketTwoBalance = useGetTreasuryBucketBalance(treasuryContracts[1].address)
   const bucketThreeBalance = useGetTreasuryBucketBalance(treasuryContracts[2].address)
   const bucketFourBalance = useGetTreasuryBucketBalance(treasuryContracts[3].address)
+  const bucketFiveBalance = useGetTreasuryBucketBalance(treasuryContracts[4].address)
```

Comment:
This is okay for now, and I know this isn't you, but I need to mention it. I don't like these variable names with indexes, the object below should not be an array, it should be an object with a unique key and we use that unique key here.

Reply:
I created #301 for this